gatsby-config.js

```diff
@@ -463,17 +463,17 @@ module.exports = {
                 title: "Identity for Edge Network",
                 path: "/documentation/identity-for-edge-network",
                 pages: [
+                    {
+                        title: "Release notes",
+                        path: "documentation/identity-for-edge-network/release-notes"
+                    },
```

Comment:
I don't think I agree with the decision to put release notes on the top - generally, at least for all the Platform docs, we have Release notes at the bottom of the section, since it's (relatively) one of the less important documents to have?

If you guys really think we should have release notes on top, I'll move it, but me having Release notes at the bottom was a decision I intentionally made.

Reply:
The order we were following was: migrating to AEP, release notes, API ref, event ref, faq, others. I am ok with moving release notes to the bottom since we also have a dedicated section for release notes at top level, and keep the order for the others.